editor/shader_create_dialog.cpp

```diff
@@ -142,31 +142,66 @@ void ShaderCreateDialog::_create_new() {
 		code += vformat("shader_type %s;\n", mode_menu->get_text().replace(" ", "").camelcase_to_underscore());
 
 		if (current_template == 0) { // Default template.
-			code += "\n";
 			switch (current_mode) {
 				case Shader::MODE_SPATIAL:
-					code += "void fragment() {\n";
-					code += "\t// Place fragment code here.\n";
-					code += "}\n";
+					// `vertex()` and `light()` are commented out by default
+					// as these are used less often.
+					// Functions are ordered in the vertex-fragment-light order
+					// to follow established conventions.
+					code += R"(
+//void vertex() {
+	// Called for every vertex the material is visible on.
+//}
```

Comment:
Should we uncomment `vertex()` by default? Does it have any performance cost if it's defined but left empty?

The same comment applies for CanvasItem shaders.

Reply:
I'm not sure is this a good idea - vertex shader is not used quite often as fragment.